src/Symfony/Component/WebLink/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "symfony/weblink",
```

Comment:
What about going back to "Link":

 * The PSR is named Link, not Weblink;
 * Short names are better;
 * No dash (that helps :));
 * Do we think that we will have a non-web link component at some point? (or do we think that Link can be confusing?)

WDYT? If everybody think that web-link is better, that's of course an option as well.